### ISSUE TITLE
Thorium on android cannot detect google account on device

### ISSUE THREAD

**Issue by 9M2PJU** [Alex313031/Thorium-Android#37]
thorium installed, first time run it cannot detect google account on my device.

**Comment by phamcuong26124** [comment]
I also have the same problem on sony xperia 1 ii

**Comment by awee1233** [comment]
I also have same problem on redmi note 10 pro

**Comment by Manfred73** [comment]
Just intalled on my Android phone (S23 Ultra), and it doesn't recognize my existing google account. I can only add an account and then it says account already exists.

**Comment by supwner** [comment]
i also have this problem in MIUI Global 14.0.3

**Comment by Facupelli** [comment]
I also have same problem on Poco X5 pro

**Comment by ndom91** [comment]
Idk if it's related, but thorium doesn't seem to correctly register as a browser, i.e. it doesn't appease as a handler for url shares and 1password doesn't recognize input fields there 🤔

**Comment by weskerty** [comment]
Same on MicroG

**Comment by rabaimor** [comment]
Same here. I even tried to add a new account, but that doesnt work either

**Comment by Magnusretrotech** [comment]
Same

**Comment by naups** [comment]
Just make it your default browser then relaunch app, thorium would see ur email there.
But it wont sync because Thorium will need to verify the account. It wont stop need verify email.

Tested using LineageOs A11 (msm8937 board)

**Comment by MulverineX** [comment]
Seems like the way to go here would be the use of https://github.com/TeamVanced/VancedMicroG. @Alex313031

**Comment by utade** [comment]
doesn't work too on Samsung A13

**Comment by animanna** [comment]
I Also have this problem. Can't login to my Google account to Sync.

**Comment by BigSmoke010** [comment]
Same

**Comment by theparthacus** [comment]
> Seems like the way to go here would be the use of https://github.com/TeamVanced/VancedMicroG. @Alex313031

yep. in fact, i would want to donate a little for this way. not everyone wants to make it their default browser.

**Comment by MulverineX** [comment]
As of late, this is now the up to date utility for that.

https://github.com/Revanced/GmsCore

**Comment by bubobih** [comment]
To confirm that login is not working, not even with microg.

**Comment by MulverineX** [comment]
@bubobih supporting an alternative means of google auth requires more work than installing microg/GmsCore. This issue is basically a suggestion for Alex to do [that work](https://github.com/ReVanced/revanced-patches/tree/main/src%2Fmain%2Fkotlin%2Fapp%2Frevanced%2Fpatches%2Fyoutube%2Fmisc%2Fgms), as otherwise its impossible for google auth to function, because stock GMS does app signature checks.

**Comment by ZeTomaz** [comment]
I also have a problem with the installation on my _**Redmi note 11 Pro 4G**_. Thorium can't detect my Google credentials.

**Comment by pro9code** [comment]
Same, "already in use"

**Comment by A81N9** [comment]
I also have same problem on redmi k30 pro

**Comment by pakav0** [comment]
It's a great browser for Windows 11, but I don't see any point in it if it doesn't sync with Android. Does Alex313031 plan to implement authentication via MicroG?

**Comment by gz83** [comment]
Third-party Chromium fork browsers are not allowed to use Google's native account login process. In the new version, I have managed to hide the relevant entry point.
